src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/PackageLibs.targets

```diff
@@ -140,6 +153,13 @@
       <FilesToPackage Condition="'%(FilesToPackage.SubFolder)' != ''">
         <TargetPath>%(TargetPath)%(FilesToPackage.SubFolder)</TargetPath>
       </FilesToPackage>
+
+      <!-- Include source files for symbol packages. -->
+      <FilesToPackage Include="@(Compile->'%(FullPath)')">
+        <TargetPath>src</TargetPath>
```

Comment:
Is there ever any case where a source file isn't somewhere under ProjectRoot? If there are is my assumption that they just end up in the root src target folder? Will the pdb indexing of the source code actually be affected by that?